src/erlcloud_sns.erl

```diff
@@ -240,14 +242,14 @@ get_endpoint_attributes(EndpointArn, Config) ->
             Doc),
     [{arn, EndpointArn} | Decoded].
 
--spec get_endpoint_attributes/3 :: (string(), string(), string()) -> sns_endpoint().
+-spec get_endpoint_attributes/3 :: (parameter(), parameter(), parameter()) -> sns_endpoint().
```

Comment:
Line 245 is too long: -spec get_endpoint_attributes/3 :: (parameter(), parameter(), parameter()) -> sns_endpoint()..